### ISSUE TITLE
Can you add a UpdatedRowsFetchSpecExtensions.kt

### ISSUE THREAD

**Issue by pull-vert** [spring-projects/spring-data-r2dbc#212]
This Kotlin file would contain a single coroutine extention of the ```rowsUpdated()``` method from UpdatedRowsFetchSpec interface, it would look like :
```kotlin
suspend fun UpdatedRowsFetchSpec.awaitRowsUpdated(): Int = ...
```

**Comment by mp911de** [comment]
Makes sense. Do you want to submit a a pull request?

**Comment by pull-vert** [comment]
> Makes sense. Do you want to submit a a pull request?

**Comment by mp911de** [comment]
Fixed via #213.
